apps/public-reference/layouts/application.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 import Head from "next/head"
-import t from "@bloom-housing/ui-components/src/helpers/translator"
+import { t } from "@bloom-housing/ui-components/src/helpers/translator"
```

Comment:
While we're changing this, probably better as:

```suggestion
import { t } from "@bloom-housing/ui-components"
```

Reply:
combined with next import statement instead